<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2018-08-16T04:43:37Z" name="Suite" finished-at="2018-08-16T04:45:06Z" duration-ms="89385">
    <groups>
    </groups>
    <test started-at="2018-08-16T04:43:37Z" name="Test" finished-at="2018-08-16T04:45:06Z" duration-ms="89385">
      <class name="com.rl.qa.testcases.FavoritesPageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.rl.qa.testcases.FavoritesPageTest@45ca843]" started-at="2018-08-16T04:43:37Z" name="setup" finished-at="2018-08-16T04:44:08Z" duration-ms="31228" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyNumberOfItems()[pri:0, instance:com.rl.qa.testcases.FavoritesPageTest@45ca843]" started-at="2018-08-16T04:44:08Z" name="verifyNumberOfItems" finished-at="2018-08-16T04:44:25Z" duration-ms="16488" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyNumberOfItems -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.rl.qa.testcases.FavoritesPageTest@45ca843]" started-at="2018-08-16T04:44:25Z" name="close" finished-at="2018-08-16T04:44:26Z" duration-ms="1156" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.rl.qa.testcases.FavoritesPageTest -->
      <class name="com.rl.qa.testcases.RegistrationPageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.rl.qa.testcases.RegistrationPageTest@757acd7b]" started-at="2018-08-16T04:44:26Z" name="setup" finished-at="2018-08-16T04:44:50Z" duration-ms="23728" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="createUser()[pri:0, instance:com.rl.qa.testcases.RegistrationPageTest@757acd7b]" started-at="2018-08-16T04:44:50Z" name="createUser" finished-at="2018-08-16T04:45:06Z" duration-ms="16049" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Ralph Lauren - Account | Ralph Lauren | 103.1.11 - controllers] but found [Sites-RalphLauren_EU-Site | Ralph Lauren | 103.1.11 - controllers]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Ralph Lauren - Account | Ralph Lauren | 103.1.11 - controllers] but found [Sites-RalphLauren_EU-Site | Ralph Lauren | 103.1.11 - controllers]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.rl.qa.testcases.RegistrationPageTest.createUser(RegistrationPageTest.java:72)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createUser -->
        <test-method is-config="true" signature="close()[pri:0, instance:com.rl.qa.testcases.RegistrationPageTest@757acd7b]" started-at="2018-08-16T04:45:06Z" name="close" finished-at="2018-08-16T04:45:06Z" duration-ms="701" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.rl.qa.testcases.RegistrationPageTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
